packages/crayons-core/src/utils/draggable.ts

```diff
@@ -0,0 +1,270 @@
+import { debounce } from '.';
```

Comment:
This debounce is being used from the lodash library. Can confirm that when this draggable util is used the entire lodash lib is not getting bundled?